### PR TITLE
Testing setup

### PR DESCRIPTION
**Pull request by harrisonywu**
@Pfdiamond13 Let me know how this all looks.

### CODE REVIEW

File: client/src/components/test/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+
+configure({ adapter: new Adapter() });
+
+// Our tests are below.
```
Comment (Pfdiamond13) [inline client/src/components/test/setupTests.js]: @harrisonywu consider removing unnecessary comments from production.

File: client/src/index.jsx [hunk 2]
```diff
@@ -1,9 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
-// create a main component
-// but first start by making a shitty component that renders
-
```
Comment (Pfdiamond13) [inline client/src/index.jsx]: @harrisonywu remove unnecessary comments prior to pushing to master.

File: client/src/index.jsx [hunk 3]
```diff
@@ -16,6 +12,6 @@ class App extends React.Component {
   }
 }
 
-ReactDOM.render(<App />, document.getElementById('app'))
+ReactDOM.render(<App />, document.getElementById('app'));
 
 export default App;
```
Comment (Pfdiamond13) [inline client/src/index.jsx]: @harrisonywu eslint will ask for a newline break at the end of a file.

File: server/server.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 const express = require('express');
 
+const morgan = require('morgan'); // Add in Morgan middleware functionality.
```
Comment (Pfdiamond13) [inline server/server.js]: @harrisonywu consider removing unnecessary comments from production.

**Comment by harrisonywu** [comment]
@Pfdiamond13 Thanks for the comments, I'll resubmit a pull request that should fix these issues and only include files related to Jest & Enzyme setup.

**Pull request closed without merging**